compiler/cpp/src/thrift/generate/t_rs_generator.cc

```diff
@@ -2668,7 +2677,7 @@ void t_rs_generator::render_sync_process_function(t_function *tfunc, const strin
     << "let "
     << (has_non_void_args(tfunc) ? "args" : "_")
     << " = "
-    << rust_struct_name(tfunc->get_arglist())
```

Comment:
Good change. Now both the `service_call_{args,result}_struct_name` functions take the same parameter 👍